developer/app-router/installation.mdx

```diff
@@ -200,3 +200,22 @@ components from the left toolbar.
 
 </Step>
 </Steps>
+
+## Next Steps
+
+Here are a couple of ideas for what you might want to do next.
+
+<CardGroup cols={3}>
+  <Card title="Concepts" href="/developer/concepts">
+    Learn core concepts about how Makeswift works.
+  </Card>
+  <Card
+    title="Custom components"
+    href="/developer/guides/how-to/built-in-components"
+  >
+    Build custom components to be used in the Visual Builder.
```

Comment:
"Build custom components for use in the Visual Builder." ?

Reply:
Updated!